napari_animation/animation.py

```diff
@@ -225,15 +225,15 @@ def animate(
         if not save_as_folder:
             try:
                 # create imageio writer and add all frames
-                if quality is not None:
+                if path_obj.suffix in ['mov', 'avi', 'mpg', 'mpeg', 'mp4', 'mkv', 'wmv']:
                     writer = imageio.get_writer(
                         path, fps=fps, quality=quality, format=format,
                     )
                 else:
```

Comment:
could you add a comment explaining the need for this if/else block, it's not obvious without knowing that gif is the else case why this is happening

Reply:
Yes that makes sense. I guess in principle the ```else``` works with anything that's accepted by imageio but not in the ffmpeg-list. But except gif I don't think there's any other possible format. I also updated the parameter definition so that it's clear that several formats other than mp4 are available.